CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Releases
 
+## v1.5.0 (November 4, 2019)
+
+### Enhancements
+
+- **Deep Dependency Graph:** Implement Service Oriented Dependency Graph (DDG) ([@tiffon](https://github.com/tiffon) in [#442](https://github.com/jaegertracing/jaeger-ui/pull/442), [#441](https://github.com/jaegertracing/jaeger-ui/pull/441), [#440](https://github.com/jaegertracing/jaeger-ui/pull/440), [#439](https://github.com/jaegertracing/jaeger-ui/pull/439), [#438](https://github.com/jaegertracing/jaeger-ui/pull/438), [#437](https://github.com/jaegertracing/jaeger-ui/pull/437), [#431](https://github.com/jaegertracing/jaeger-ui/pull/431), [#406](https://github.com/jaegertracing/jaeger-ui/pull/406), and [#405](https://github.com/jaegertracing/jaeger-ui/pull/405); and [@everett980](https://github.com/everett980) in [#468](https://github.com/jaegertracing/jaeger-ui/pull/468), [#466](https://github.com/jaegertracing/jaeger-ui/pull/466), [#458](https://github.com/jaegertracing/jaeger-ui/pull/458), [#463](https://github.com/jaegertracing/jaeger-ui/pull/463), [#457](https://github.com/jaegertracing/jaeger-ui/pull/457), [#453](https://github.com/jaegertracing/jaeger-ui/pull/453), [#452](https://github.com/jaegertracing/jaeger-ui/pull/452), [#434](https://github.com/jaegertracing/jaeger-ui/pull/434), [#433](https://github.com/jaegertracing/jaeger-ui/pull/433), [#432](https://github.com/jaegertracing/jaeger-ui/pull/432), [#429](https://github.com/jaegertracing/jaeger-ui/pull/429), [#427](https://github.com/jaegertracing/jaeger-ui/pull/427), [#424](https://github.com/jaegertracing/jaeger-ui/pull/424), [#421](https://github.com/jaegertracing/jaeger-ui/pull/421), [#412](https://github.com/jaegertracing/jaeger-ui/pull/412), [#398](https://github.com/jaegertracing/jaeger-ui/pull/398), [#386](https://github.com/jaegertracing/jaeger-ui/pull/386), and [#378](https://github.com/jaegertracing/jaeger-ui/pull/378))
+
+- **Deep Dependency Graph:** Derive DDG from search results ([@rubenvp8510](https://github.com/rubenvp8510) in [#445](https://github.com/jaegertracing/jaeger-ui/pull/445))
+
+- **Configuration:** Add script tags from config to body ([@everett980](https://github.com/everett980) in [#455](https://github.com/jaegertracing/jaeger-ui/pull/455))
```

Comment:
can we rephrase this? without looking into the ticket I don't know what this means